flutter/lib/src/sentry_flutter.dart

```diff
@@ -10,34 +9,36 @@ import 'package:sentry/sentry.dart';
 import 'default_integrations.dart';
 import 'file_system_transport.dart';
 import 'version.dart';
+import 'web_platform_checker.dart' if (dart.library.io) 'platform_checker.dart';
 
 /// Sentry Flutter SDK main entry point
 mixin SentryFlutter {
   static const _channel = MethodChannel('sentry_flutter');
 
   static Future<void> init(
     OptionsConfiguration optionsConfiguration,
-    Function callback, {
+    AppRunner callback, {
```

Comment:
```suggestion
    [AppRunner appRunner], {
```

Reply:
✅ I had to add it to optional named arguments